pkg/policy/api/rule.go

```diff
@@ -411,14 +413,23 @@ type PortRuleHTTP struct {
 // optional, if all fields are empty or missing, the rule will match all
 // Kafka messages.
 type PortRuleKafka struct {
-	// APIVersion is the version matched against the api version of the
-	// Kafka message. If set, it has to be a string representing a positive
-	// integer.
+	// Role is a case-insensitive string and describes a group of API keys
+	// necessary to perform certain higher level Kafka operations such as "produce"
+	// or "consume". An APIGroup automatically expands into all APIKeys required
+	// to perform the specified higher level operation.
 	//
-	// If omitted or empty, all versions are allowed.
+	// The following values are supported:
+	//  - "produce": Allow producing to the topics specified in the rule
+	//  - "consume": Allow consuming from the topics specified in the rule
+	//
+	// This field is incompatible with the APIKey field, either APIKey or Role
+	// may be specified.
+	//
+	// If omitted or empty, the field has no effect and the logic of the APIKey
+	// field applies.
 	//
 	// +optional
-	APIVersion string `json:"apiVersion,omitempty"`
+	Role string `json:"Role,omitempty"`
```

Comment:
All other JSON fields are lower case, let's change it to `role`

Reply:
done